pkg/kubernetes/watch/watcher_test.go

```diff
@@ -497,7 +497,7 @@ func TestWatcher(t *testing.T) {
 
 			return nil
 		}
-		second = func(opts metav1.ListOptions) error {
+		second = func(opts metav1.ListOptions) (unused error) { // nolint:unparam
```

Comment:
Maybe `(_ error)`?

Reply:
I've opted to just add a comment to the nolint directive.